<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserQuestions</class>
 <widget class="QDialog" name="UserQuestions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 12pt &quot;TakaoPGothic&quot;;</string>
  </property>
  <widget class="QPlainTextEdit" name="planArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="questionsButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Questions</string>
   </property>
  </widget>
  <widget class="QPushButton" name="visualiseButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualise</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>379</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="question_label3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>371</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;TakaoPGothic&quot;;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>311</width>
       <height>135</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="addOption">
        <property name="text">
         <string>Why action A is not involved in the plan?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="removeOption">
        <property name="text">
         <string>Why action A is involved in the plan?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="replaceOption">
        <property name="text">
         <string>Why action A rather than action B? </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rescheduleOption">
        <property name="text">
         <string>Why action A at this time? </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>309</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Do you want to know:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="statementLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>List of actions:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QListWidget" name="list">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="doneButton">
        <property name="text">
         <string>List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="completeButton">
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Is this your question? </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="question_label">
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;TakaoPGothic&quot;;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>go back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;TakaoPGothic&quot;;</string>
   </property>
   <property name="text">
    <string>Select one of the following question and click &quot;List&quot;:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
